packages/desktop-client/src/components/manager/WelcomeScreen.js

```diff
@@ -26,17 +26,19 @@ function WelcomeScreen({ createBudget, pushModal }) {
           top of that, we want to provide powerful tools to allow you to do
           whatever you want with your data.
         </P>
-        <P style={{ '& a, & a:visited': { color: colors.p5 } }}>
+        <P>
           Currently, Actual implements budgeting based on a{' '}
           <ExternalLink
-            asAnchor
-            style={{ color: colors.p5 }}
-            href="https://actualbudget.org/docs/budgeting/"
+            to="https://actualbudget.org/docs/budgeting/"
+            linkColor="purple"
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** would be better to use `primary`/`secondary`/`tertiary`/`muted` etc. names here. Currently the primary color is purple. Will it be the same forever? Not sure. Will it be the same for darkmode? Also not sure.

Reply:
This is very color specific at the moment. I noticed that typically blue is used on white backgrounds while purple is used on blue backgrounds.